packages/react-components/react-motion-components-preview/library/src/components/Collapse/Collapse.ts

```diff
@@ -43,7 +43,19 @@ export const createCollapsePresence: PresenceMotionFnCreator<CollapseVariantPara
     const sizeName = orientation === 'horizontal' ? 'maxWidth' : 'maxHeight';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33063&commitId=73daa73d7b74346debc890992b762dade9cfb47f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|2| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|3279| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>